### PR TITLE
Integrate Sentry

### PR DESCRIPTION
**Pull request by RaitoBezarius**
Add Sentry integration for Mangaki.

- ~Untested on Vagrant.~ (we don't want this on Vagrant, anyway).
- :x: Untested on a real machine.
- :white_check_mark: Locally tested.

Uses https://sentry.io service for https://beta.mangaki.fr.

### CODE REVIEW

File: mangaki/mangaki/settings.py [hunk 1]
```diff
@@ -139,10 +152,43 @@
...
+            'level': 'ERROR',
+            'class': 'raven.contrib.django.raven_compat.handlers.SentryHandler',
+    }
+    LOGGING['root']['handlers'].append('sentry')
+    LOGGING['loggers']['raven'] = {
+        'level': 'DEBUG',
+        'handlers': ['console'],
+        'propagate': False,
+    }
+    LOGGING['loggers']['sentry.errors'] = {
```
Comment (Elarnon) [inline mangaki/mangaki/settings.py]: This is already the value of that entry.

File: mangaki/mangaki/settings.py [hunk 2]
```diff
@@ -139,10 +152,43 @@
...
     },
 }
 
+if config.has_section('sentry'):
+    LOGGING['handlers']['sentry'] = {
+            'level': 'ERROR',
+            'class': 'raven.contrib.django.raven_compat.handlers.SentryHandler',
+    }
+    LOGGING['root']['handlers'].append('sentry')
+    LOGGING['loggers']['raven'] = {
```
Comment (Elarnon) [inline mangaki/mangaki/settings.py]: This is already the value of that entry.

File: provisioning/roles/mangaki_source/templates/settings.ini.j2 [hunk 3]
```diff
@@ -26,6 +26,11 @@ DB_HOST = {{ mangaki_source_db_host }}
 DB_NAME = {{ mangaki_source_db_name }}
 DB_USER = {{ mangaki_source_db_user }}
 
+{% if mangaki_source_has_sentry | bool %}
```
Comment (Elarnon) [inline provisioning/roles/mangaki_source/templates/settings.ini.j2]: Please write this as `{% if mangaki_source_has_sentry|bool %}` for consistency.

File: mangaki/mangaki/settings.py (hunk 2 above)
Comment (RaitoBezarius) [inline mangaki/mangaki/settings.py]: Epic fail. Thank you.

**Comment by RaitoBezarius** [comment]
Please review again, @Elarnon.

**Comment by RaitoBezarius** [comment]
@Elarnon Let's just wait for a chance to test this on `beta` before merge.

**Pull request merged**